### PR TITLE
Uk government bank holidays ipc

### PR DESCRIPTION
**Pull request by martynlesbirel**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

File: independent-publisher-connectors/U.K. Government Bank Holidays/UK-Bank-Holidays.swagger.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "UK Bank Holidays",
+    "description": "A JSON object containing the bank holidays of the four kingdoms of the United Kingdom supplied by H.M. Government",
+    "version": "1.0",
+    "contact": {
+      "name": "Martyn Lesbirel",
+      "email": "ipc-author-bh@outlook.com"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/U.K. Government Bank Holidays/UK-Bank-Holidays.swagger.json:9]: @martynlesbirel Please add `url` property as well

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/U.K. Government Bank Holidays/UK-Bank-Holidays.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply independent-publisher-connectors/U.K. Government Bank Holidays/UK-Bank-Holidays.swagger.json:9]: @martynlesbirel Please add url property as well

Please let us know if you do not have one.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

File: independent-publisher-connectors/U.K. Government Bank Holidays/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Martyn Lesbirel",
+    "website": "https://www.gov.uk/bank-holidays",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/U.K. Government Bank Holidays/apiProperties.json:7]: @martynlesbirel Can you please remove website property.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by mamurshe — APPROVED** [review]

**Pull request merged**